frontend/apps/filemanager/filemanagercollection.lua

```diff
@@ -74,14 +75,14 @@ end
 
 function FileManagerCollection:onShowColl(collection_name)
     collection_name = collection_name or ReadCollection.default_collection_name
-    self.coll_menu = BookList:new{
+    self.booklist_menu = BookList:new{
```

Comment:
I think this change of name everywhere is because you access `this.booklist_menu` in `CoverBrowser.getUpdateItemTableFunc()` and you need that same name everywhere.
I think it should be mentionned as a comment here (and elsewhere).
`-- This may be hijacked by CoverBrowser pluging and needs to be known as booklist_menu.`
(or something like that if I'm not fully correct with that comment).